### PR TITLE
[HOPSWORKS-1405] Register livy server with consul

### PR DESCRIPTION
**Pull request by kouzant**
https://logicalclocks.atlassian.net/browse/HOPSWORKS-1405

### CODE REVIEW

File: templates/default/livy-health.sh.erb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+set -e
+curl --fail -X GET "http://$HOSTNAME:<%= node['livy']['port'] %>/sessions?from=0&size=1"
```
Comment (tkakantousis) [inline templates/default/livy-health.sh.erb]: https://superuser.com/a/657174 maybe `--fail` is not that safe?

Comment (kouzant) [reply templates/default/livy-health.sh.erb]: I'm not sure where the `But it blocks output to the screen.` comes from. I've never experienced such issue. I can add the `--silent` flag but I don't see why `--fail` is not safe

Comment (tkakantousis) [reply templates/default/livy-health.sh.erb]: Let's keep it as it is then

Comment (kouzant) [reply templates/default/livy-health.sh.erb]: Actually I have forgotten to redirect stdout and stderr to /dev/null So I'm adding the `--silent` flag and redirection of outputs ;)

**Pull request merged**